tools/reviewer/webapp/src/app/dashboard/file-changes/services/commit.service.ts

```diff
@@ -48,6 +54,15 @@ export class CommitService {
 
     this.addCommitId(this.stateService.leftCommitId);
     this.addCommitId(this.stateService.rightCommitId);
+
+    // Switch left and right commits, if left commit id newer than right one
```

Comment:
Can this happen through the UI, or just through the query params?

Reply:
Only by query params